internal/storage/storageutil/object_attrs.go

```diff
@@ -0,0 +1,58 @@
+package storageutil
+
+import (
+	"crypto/md5"
+	"time"
+
+	"cloud.google.com/go/storage"
+	"github.com/jacobsa/gcloud/gcs"
+	storagev1 "google.golang.org/api/storage/v1"
+)
+
+func ObjectAttrsToBucketObject(attrs *storage.ObjectAttrs) *gcs.Object {
+	// Converting []ACLRule returned by the Go Client into []*storagev1.ObjectAccessControl which complies with GCSFuse type.
+	var acl []*storagev1.ObjectAccessControl
+	for _, element := range attrs.ACL {
+		currACL := &storagev1.ObjectAccessControl{
+			Entity:   string(element.Entity),
+			EntityId: element.EntityID,
+			Role:     string(element.Role),
+			Domain:   element.Domain,
+			Email:    element.Email,
+			ProjectTeam: &storagev1.ObjectAccessControlProjectTeam{
+				ProjectNumber: element.ProjectTeam.ProjectNumber,
+				Team:          element.ProjectTeam.Team,
+			},
+		}
+		acl = append(acl, currACL)
+	}
+
+	// Converting MD5[] slice to MD5[md5.Size] type fixed array as accepted by GCSFuse.
+	var MD5 [md5.Size]byte
```

Comment:
here as well. all variables should start with small letter and follow camelCase

Reply:
fixed